libproc/proc.c

```diff
@@ -99,10 +106,27 @@ _open (pctx_t ctx)
 int
 proc_open (pctx_t ctx, const char *path)
 {
+    struct stat buf;
+    int rc;
```

Comment:
"rc" doesn't seem to actually do anything, but that is super minor.

Reply:
Agreed.